scripts/globals/atma.lua

```diff
@@ -2,817 +2,185 @@
+    [dsp.ki.ATMA_OF_THE_TWIN_CLAW]              = {dsp.mod.DEF, 40, dsp.mod.MDEF, 20, dsp.mod.CHARMRES, 20},
+    [dsp.ki.ATMA_OF_ALLURE]                     = {dsp.mod.MPP, 30, dsp.mod.MND, 30, dsp.mod.ENMITY, -30},
+    [dsp.ki.ATMA_OF_ETERNITY]                   = {dsp.mod.ENEMYCRITRATE, -20, dsp.mod.SLOWRES, 30, dsp.mod.CURSERES, 30},
+    [dsp.ki.ATMA_OF_THE_HEAVENS]                = {dsp.mod.MACC, 30, dsp.mod.DMGPHYS, -10, dsp.mod.PARALYZERES, 30},
+    [dsp.ki.ATMA_OF_THE_BAYING_MOON]            = {dsp.mod.ATT, 30, dsp.mod.MATT, 30},
+    [dsp.ki.ATMA_OF_THE_EBON_HOOF]              = {dsp.mod.HPP, 30, dsp.mod.SLEEPRES, 50},
+    [dsp.ki.ATMA_OF_TREMORS]                    = {dsp.mod.DMG, -20, dsp.mod.SILENCERES, 40},
+    [dsp.ki.ATMA_OF_THE_SAVAGE_TIGER]           = {dsp.mod.AGI, 30, dsp.mod.DOUBLE_ATTACK, 10},
+    [dsp.ki.ATMA_OF_THE_VORACIOUS_VIOLET]       = {dsp.mod.STR, 50, dsp.mod.DOUBLE_ATTACK, 10, dsp.mod.REGAIN, 20},
+    [dsp.ki.ATMA_OF_CLOAK_AND_DAGGER]           = {dsp.mod.ACC, 40, dsp.mod.EVA, 40},
+    [dsp.ki.ATMA_OF_THE_STORMBIRD]              = {dsp.mod.ACC, 40, dsp.mod.THUNDERATT, 40, dsp.mod.REFRESH, 5},
+    [dsp.ki.ATMA_OF_THE_NOXIOUS_FANG]           = {dsp.mod.SUBTLE_BLOW, 40, dsp.mod.WATERATT, 40, dsp.mod.POISONRES, 40},
+    [dsp.ki.ATMA_OF_VICISSITUDE]                = {dsp.mod.DEF, 40, dsp.mod.MDEF, 20, dsp.mod.REGEN, 15},
+    [dsp.ki.ATMA_OF_THE_BEYOND]                 = {dsp.mod.MATT, 30, dsp.mod.ICEATT, 30, dsp.mod.LIGHTATT, 30},
+    [dsp.ki.ATMA_OF_STORMBREATH]                = {dsp.mod.VIT, 30, dsp.mod.DMGBREATH, -30},
+    [dsp.ki.ATMA_OF_GALES]                      = {dsp.mod.WINDATT, 30, dsp.mod.WINDACC, 30},
+    [dsp.ki.ATMA_OF_THRASHING_TENDRILS]         = {dsp.mod.CHR, 30, dsp.mod.CRITHITRATE, 20},
+    [dsp.ki.ATMA_OF_THE_DRIFTER]                = {dsp.mod.RATT, 30, dsp.mod.RACC, 40},
+    [dsp.ki.ATMA_OF_THE_STRONGHOLD]             = {dsp.mod.ATT, 40, dsp.mod.DEF, 40, dsp.mod.REGEN, 15},
+    [dsp.ki.ATMA_OF_THE_HARVESTER]              = {dsp.mod.STR, 30, dsp.mod.DOUBLE_ATTACK, 10, dsp.mod.SLEEPRES, 40},
+    [dsp.ki.ATMA_OF_DUNES]                      = {dsp.mod.STORETP, 20, dsp.mod.SLOWRES, 40},
+    [dsp.ki.ATMA_OF_THE_COSMOS]                 = {dsp.mod.DARKATT, 40, dsp.mod.AMNESIARES, 40, dsp.mod.SILENCERES, 40},
+    [dsp.ki.ATMA_OF_THE_SIREN_SHADOW]           = {dsp.mod.ATT, 40, dsp.mod.EVA, 40, dsp.mod.PARALYZERES, 40},
+    [dsp.ki.ATMA_OF_THE_IMPALER]                = {dsp.mod.DOUBLE_ATTACK, 20, dsp.mod.BINDRES, 40, dsp.mod.BLINDRES, 40},
+    [dsp.ki.ATMA_OF_THE_ADAMANTINE]             = {dsp.mod.VIT, 20, dsp.mod.DEF, 40},
+    [dsp.ki.ATMA_OF_CALAMITY]                   = {dsp.mod.SLOWRES, 40, dsp.mod.BLINDRES, 40},
+    [dsp.ki.ATMA_OF_THE_CLAW]                   = {dsp.mod.EARTHATT, 30, dsp.mod.EARTHACC, 40},
+    [dsp.ki.ATMA_OF_BALEFUL_BONES]              = {dsp.mod.STR, 20, dsp.mod.DARKACC, 40)
```

Comment:
looks like a typo here
```lua
{dsp.mod.STR, 20, dsp.mod.DARKACC, 40)
```
->
```lua
{dsp.mod.STR, 20, dsp.mod.DARKACC, 40},
```

Reply:
Huh.  Wonder how that one got through find/replace.  Fixed.